README.md

```diff
@@ -1,31 +1,32 @@
 # Native LangChain4j Searchbot
 
-This is a Java command-line chatbot application that leverages [LangChain4j](https://github.com/langchain4j/langchain4j) and the OPENAI API model "gpt-4". 
+This is a Java command-line chatbot example application that leverages the [LangChain4j](https://github.com/langchain4j) API and version [_GPT-4_ of OpenAI](https://openai.com/index/gpt-4/). 
 
-> [LangChain4j](https://github.com/langchain4j/langchain4j) is a Java version of LangChain to simplify integrating AI/LLM capabilities into Java applications.
+> LangChain4j is a Java implementation of [LangChain](https://www.langchain.com/) whose goal is to simplify integrating AI/LLM capabilities into Java applications.
 
-The example is based on a [LangChain4j tutorial](:https://github.com/langchain4j/langchain4j-examples).
-This application is enhanced with [GraalVM Native Image Maven plugin](https://graalvm.github.io/native-build-tools/latest/index.html) that enables you to easily create a native executable for a Java application.
+The example is based on a [LangChain4j tutorial](https://github.com/langchain4j/langchain4j-examples).
```

Comment:
This one https://github.com/langchain4j/langchain4j-examples/blob/main/open-ai-examples/src/main/java/OpenAiChatModelExamples.java